app/core/language.js

```diff
@@ -9,64 +9,88 @@ import mainFR from '../../locales/fr/translation.json';
 import parameterFR from '../../locales/fr/parameter.json';
 import mainDE from '../../locales/de/translation.json';
 import parameterDE from '../../locales/de/parameter.json';
+import mainIT from '../../locales/it/translation.json';
 
-// Update moment with the right language, for date display
-i18n.on('languageChanged', lng => {
-  moment.locale(lng);
-  if (self.localStorage) {
-    self.localStorage.lang = lng;
-  }
-});
+const crowdinActive = typeof _jipt === 'object';
 
-let defaultLanguage = getLocale();
+let language = getLocale();
 if (self.localStorage && self.localStorage.lang) {
-  defaultLanguage = self.localStorage.lang;
+  language = self.localStorage.lang;
 }
 
-i18n
-  .use(reactI18nextModule)
-  .init({
-    fallbackLng: 'en',
-    lng: defaultLanguage,
+const i18nOptions = {
+  fallbackLng: 'en',
+  lng: language,
 
-    // To allow . in keys
-    keySeparator: false,
-    // To allow : in keys
-    nsSeparator: ':::',
+  // To allow . in keys
+  keySeparator: false,
+  // To allow : in keys
+  nsSeparator: ':::',
 
-    debug: false,
+  debug: false,
 
-    interpolation: {
-      escapeValue: false, // not needed for react!!
-    },
+  interpolation: {
+    escapeValue: false, // not needed for react!!
+  },
+
+  // If the translation is empty, return the key instead
+  returnEmptyString: false,
 
-    // If the translation is empty, return the key instead
-    returnEmptyString: false,
+  react: {
+    wait: true,
+    withRef: true,
+    defaultTransParent: 'div', // a valid react element - required before react 16
+    transSupportBasicHtmlNodes: true, // allow <br/> and simple html elements in translations
+  },
+  ns: ['main', 'params'],
+  defaultNS: 'main',
 
-    react: {
-      wait: true,
-      withRef: true,
-      defaultTransParent: 'div', // a valid react element - required before react 16
-      transSupportBasicHtmlNodes: true, // allow <br/> and simple html elements in translations
+  resources: {
+    en: {
+      // Default namespace
+      main: mainEN,
+      params: parameterEN
+    },
+    fr: {
+      main: mainFR,
+      params: parameterFR
     },
-    ns: ['main', 'params'],
-    defaultNS: 'main',
-
-    resources: {
-      en: {
-        // Default namespace
-        main: mainEN,
-        params: parameterEN
-      },
-      fr: {
-        main: mainFR,
-        params: parameterFR
-      },
-      de: {
-        main: mainDE,
-        params: parameterDE
-      }
+    de: {
+      main: mainDE,
+      params: parameterDE
+    }
+  }
+}
+
+
+i18n.use(reactI18nextModule);
+
+if (crowdinActive) {
+  i18nOptions.fallbackLng = 'it';
```

Comment:
Why use Italian as a default language ?

Reply:
italian is a fake language used for crowdin live preview ;)